### DISCUSSION TITLE
What is widget custom subtitle keyword for Air qulity?

### DISCUSSION THREAD

**Discussion by hyunki** [breezy-weather/breezy-weather#412] [Q&A] [answered]
I want my widget representing
CUV
Air quality

But I can't find keword for air quality on description.
"aqi" wasn't one.

I hope there is custom widget editing in text widget view.

**Comment by papjul** [comment, marked as answer]
`$caqi$` is current air quality, and `$0aqi$` the daily air quality with 0 = today, 1 = tomorrow, and so on
